lua/core/gitsigns.lua

```diff
@@ -1,63 +1,21 @@
-local M = {}
-
-M.config = function()
-  lvim.builtin.gitsigns = {
+local M = {
+  defaults = {
     active = true,
     on_config_done = nil,
-    opts = {
-      signs = {
-        add = {
-          hl = "GitSignsAdd",
-          text = "▎",
-          numhl = "GitSignsAddNr",
-          linehl = "GitSignsAddLn",
-        },
-        change = {
-          hl = "GitSignsChange",
-          text = "▎",
-          numhl = "GitSignsChangeNr",
-          linehl = "GitSignsChangeLn",
-        },
-        delete = {
-          hl = "GitSignsDelete",
-          text = "契",
-          numhl = "GitSignsDeleteNr",
-          linehl = "GitSignsDeleteLn",
-        },
-        topdelete = {
-          hl = "GitSignsDelete",
-          text = "契",
-          numhl = "GitSignsDeleteNr",
-          linehl = "GitSignsDeleteLn",
-        },
-        changedelete = {
-          hl = "GitSignsChange",
-          text = "▎",
-          numhl = "GitSignsChangeNr",
-          linehl = "GitSignsChangeLn",
-        },
-      },
-      numhl = false,
-      linehl = false,
-      keymaps = {
-        -- Default keymap options
-        noremap = true,
-        buffer = true,
-      },
-      watch_index = { interval = 1000 },
-      sign_priority = 6,
-      update_debounce = 200,
-      status_formatter = nil, -- Use default
-    },
-  }
+    config = {},
```

Comment:
Why did this suddenly get deleted?

Reply:
I'm sorry if these were not defaults

Reply:
I double-checked, only the signs are different:
- our: https://github.com/LunarVim/LunarVim/blob/master/lua/core/gitsigns.lua#L8
- defaults: https://github.com/lewis6991/gitsigns.nvim/blob/main/lua/gitsigns/config.lua#L83

I think the defaults are better, I wonder if ours are old defaults or if we decided to have them.
Anyway, I'm obviously open to modifying them.